socorro/scripts/es.py

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Elasticsearch manipulation script for deleting and creating Elasticsearch
+indices.
+"""
+
+import argparse
+import datetime
+import os
+
+from configman import configuration, Namespace
+
+from socorro.external.es.base import generate_list_of_indexes
+from socorro.external.es.connection_context import ConnectionContext
+from socorro.scripts import WrappedTextHelpFormatter
+
+
+DESCRIPTION = 'Create and delete Elasticsearch indices'
+
+EPILOG = 'Requires Elasticsearch configuration to be set in environment.'
+
+
+def get_conn():
+    ns = Namespace()
+    ns.add_option('elasticsearch_class', default=ConnectionContext)
+
+    config = configuration(
+        definition_source=ns,
+        values_source_list=[{
+            'elasticsearch_urls': os.environ.get('ELASTICSEARCH_URLS', 'http://localhost:9200')
+        }]
+    )
+
+    return ConnectionContext(config)
```

Comment:
So, this kind of hard-codes how we create `ConnectionContext` and makes yet another place where we do this in addition to processor, crontabber, and the webapp. I'm not really sure what to do about that, but I didn't want this script to reach into those places and pull that stuff out.

I theorize that when we swap configman out for whatever's next, it'll be more manageable.

Reply:
The last commit fixed this. It will correctly use the defaults defined in `ConnectionContext` and that's better.